web/frontend/javascripts/check_results.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { get } from 'axios';
+
+import Accordion from '@components/Accordion';
+import ChecksTable from '@components/ChecksTable';
+
+const clusterId = window.location.pathname.split('/').pop();
+
+const groupChecks = (checks) => {
+  const groups = Object.keys(checks)
+    .map((key) => checks[key])
+    .reduce((accumulator, current) => {
+      const { group } = current;
+      return accumulator[group]
+        ? { ...accumulator, [group]: [...accumulator[group], current] }
+        : { ...accumulator, [group]: [current] };
+    }, {});
+
+  return Object.keys(groups).map((key) => {
+    return { name: key, checks: groups[key] };
+  });
+};
+
+const ClustersChecks = ({ clusterId }) => {
+  const [results, setResults] = useState([]);
+
+  useEffect(() => {
+    get(`/api/clusters/${clusterId}/results`).then(({ data: { checks } }) => {
+      const groupedChecks = groupChecks(checks);
+      console.log(groupedChecks);
```

Comment:
`console.log` leftover

Shouldn't we catch this with the linter?

Reply:
I'm gonna add the rule to the linter, you're right